content/chat/rooms/occupancy.textile

```diff
@@ -12,7 +12,7 @@ languages:
 Occupancy enables you to view the number of users currently online in a room. This feature can be used to display user counts to highlight popular, or trending chat rooms.
 
 <aside data-type='note'>
-<p>Occupancy needs to be "enabled":/docs/chat/rooms#create when creating or retrieving a room.</p>
+<p>As Occupancy incurs additional costs, it is disabled by default and needs to be "enabled":/docs/chat/rooms#create when creating or retrieving a room.</p>
```

Comment:
Can we provide some additional context about the # of messages occupancy can produce so that the extra message costs can be considered?